contrib/test_jenkins.sh

```diff
@@ -527,6 +546,7 @@ run_tests() {
 	export UCX_ERROR_MAIL_FOOTER=$JOB_URL/$BUILD_NUMBER/console
 
 	do_distributed_task 0 4 build_icc
+	do_distributed_task 0 4 build_clang
```

Comment:
pls use "3 4" instead of "0 4" to distribute the tasks more equally

Reply:
updated